lua/lua-luacheck/Portfile

```
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           luarocks 1.0
+PortGroup           select 1.0
+
+luarocks.branches   5.1 5.2 5.3
+luarocks.setup      luacheck 0.24.0-2
+revision            0
+license             MIT
+categories          lua
+maintainers         @gpanders openmaintainer
+
+description         A tool for linting and static analysis of Lua code
+long_description    Luacheck is a static analyzer and a linter for Lua. \
+                    Luacheck detects various issues such as usage of undefined \
+                    global variables, unused variables and values, accessing \
+                    uninitialized variables, unreachable code and more. Most \
+                    aspects of checking are configurable: there are options \
+                    for defining custom project-related globals, for selecting \
+                    set of standard globals (version of Lua standard library), \
+                    for filtering warnings by type and name of related variable, \
+                    etc. The options can be used on the command line, put into a \
+                    config or directly into checked files as Lua comments.
+
+homepage            https://github.com/mpeterv/luacheck
+
+checksums           rmd160  3c90d231e26cd8a587c9868ab894263ba4f13afd \
+                    sha256  58eb0ab4668ad2f17947317afccd075255576bb25c61215424a82e066d4dc046 \
+                    size    224423
+
+if {${subport} ne ${name}} {
+    depends_lib-append \
+                        port:lua${luarocks.suffix}-ansicolors \
+                        port:lua${luarocks.suffix}-argparse \
+                        port:lua${luarocks.suffix}-checks \
+                        port:lua${luarocks.suffix}-luafilesystem \
+                        port:lua${luarocks.suffix}-metalua-parser
+
+    depends_run-append \
+                        port:luacheck_select
+
+    select.entries      [list ${luarocks.module} ${subport} ${subport}]
+
+    notes "
+To make this the default luacheck (i.e., the version run by \
```

Avoid double space in notes output:
```suggestion
To make this the default luacheck (i.e., the version run by\
```